easy_thumbnails/tests/test_animated_formats.py

```diff
@@ -81,3 +83,11 @@ def test_background(self):
         # indeed processed?
         self.assertEqual(frames_count, processed_frames_count)
         self.assertEqual(processed.size, (1000, 1800))
+
+    def test_gif_with_mode_p(self):
+        with open("easy_thumbnails/tests/files/animated_mode_p.gif", "rb") as im:
+            t = get_thumbnailer(im, "easy_thumbnails/tests/files/animated_mode_p.gif")
+            # Should not fail because of wrong mode and should still be animated.
+            # https://github.com/SmileyChris/easy-thumbnails/issues/653
+            thumbnail = t.get_thumbnail({'size': (500, 50), 'crop': True})
+            self.assertTrue(getattr(thumbnail.image, "is_animated", False))
```

Comment:
shouldn't it remain "animated"?

Reply:
haha, yes. sorry, too early :)

Reply:
No worries... I usually need a strong coffee in the morning before I'm ready to go. :smile: 

I'll also change the code to use `to_return.is_animated` since GIFs will always have this property.